src/sesclient.native/sesclient.native.cpp

```diff
@@ -3,89 +3,167 @@
 
 void ShowUsage(const char* exeName)
 {
-    std::cout << exeName << ":" << std::endl;
-    std::cout << "Contacts the specified Azure Batch software entitlement server to verify the provided token." << std::endl;
-    std::cout << "Parameters:" << std::endl;
-    std::cout << "    --url <software entitlement server URL>" << std::endl;
-    std::cout << "    --thumbprint <thumbprint of a certificate expected in the server's SSL certificate chain>" << std::endl;
-    std::cout << "    --common-name <common name of the certificate with the specified thumbprint>" << std::endl;
-    std::cout << "    --token <software entitlement token to pass to the server>" << std::endl;
-    std::cout << "    --application <name of the license ID being requested>" << std::endl;
+	std::cout << exeName << ":" << std::endl;
+	std::cout << "Contacts the specified Azure Batch software entitlement server to verify the provided token." << std::endl;
+	std::cout << "Parameters:" << std::endl;
+	std::cout << "    --url <software entitlement server URL>" << std::endl;
+	std::cout << "    --thumbprint <thumbprint of a certificate expected in the server's SSL certificate chain (optional)>" << std::endl;
+	std::cout << "    --common-name <common name of the certificate with the specified thumbprint (optional)>" << std::endl;
+	std::cout << "    --token <software entitlement token to pass to the server>" << std::endl;
+	std::cout << "    --application <name of the license ID being requested>" << std::endl;
```

Comment:
You can write this as:

```c++
std::cout
    << "foo" << std::endl 
    << "bar" << std::endl
    << "baz" << std::endl;
```
... to reduce the duplication of `std::cout` on all lines.

Also one other thing, if usage wasn't explicitly requested via `--help`, then we should print to `stderr` (`std::cerr`).